addons/calendar/tests/test_event_notifications.py

```diff
@@ -45,6 +45,41 @@ def test_message_invite_allday(self):
                 'partner_ids': [(4, self.partner.id)],
             }])
 
+    def test_message_invite_email_notif_mass_instant(self):
+        """Check that less than 21 notified attendees means mails are sent instantly."""
+        additional_attendees = self.env['res.partner'].create([{
+            'name': f'test{n}',
+            'email': f'test{n}@example.com'} for n in range(20)])
+        with self.mock_mail_gateway(mail_unlink_sent=False), self.mock_mail_app():
+            self.event.partner_ids = additional_attendees
+
+        self.assertNotified(
+            self._new_msgs,
+            [{
+                'is_read': True,
+                'partner': partner,
+                'type': 'email',
+            } for partner in additional_attendees],
+        )
+        self.assertEqual(self._new_msgs.mail_ids.mapped('state'), ['sent' for _ in range(20)])
+
+    def test_message_invite_email_notif_mass_queued(self):
```

Comment:
There are already tests for notifications, can't we check that standard notifications have emails sent (aka under limit) and add just one test for using the email queue when above the limit ?